test/stream/api/sink/list-001-empty-ok/003-list-sink/expected-response.json

```diff
@@ -0,0 +1,12 @@
+{
+  "projectId": %%TEST_KBC_PROJECT_ID%%,
+  "branchId": %%TEST_DEFAULT_BRANCH_ID%%,
+  "sourceId": "my-source",
+  "page": {
+    "limit": 100,
+    "totalCount": 0,
+    "sinceId": "",
+    "lastId": ""
+  },
+  "sinks": []
```

Comment:
Why is `sink` empty if we said that is always have `file` and `slice` prealocated? Or did I missed something?